ext/curl/tests/curl_unescape_basic001.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+curl_unescape basic test
+--SKIPIF--
+<?php if(!extension_loaded('curl')) print 'skip'; ?>
+--FILE--
+<?php
+
+$ch = curl_init();
+var_dump(curl_unescape($ch, '%20%21%22%23%24%25%26%27%28%29%2A%2B%2C%2D%2E%2F%30%31%32%33%34%35%36%37%38%39%3A%3B%3C%3D%3E%3F%40%41%42%43%44%45%46%47%48%49%4A%4B%4C%4D%4E%4F%50%51%52%53%54%55%56%57%58%59%5A%5B%5D%5E%5F%60%61%62%63%64%65%66%67%68%69%6A%6B%6C%6D%6E%6F%70%71%72%73%74%75%76%77%78%79%7A%7B%7C%7D%7E'));
+
+?>
+--EXPECTF--
```

Comment:
`--EXPECTF--` is for formatted result matching. It looks like you should simply be using `--EXPECT--`.